src/main/java/com/odysseusinc/athena/service/impl/VocabularyServiceImpl.java

```diff
@@ -249,7 +250,7 @@ public void restoreDownloadBundle(long bundleId) {
         if (!downloadBundle.isArchived()) {
             return;
         }
-        downloadBundleService.validate(downloadBundle);
+        downloadBundleService.validate(downloadBundle, Collections.emptyList());
```

Comment:
Ideally the list of identifiers should correspond to the one which relates to the Bundle to be restored